### PR TITLE
Release Redis 0.1.0

### PR DESCRIPTION
**Pull request by crwilcox**

### CODE REVIEW

File: docs/manifest.json [hunk 1]
```diff
@@ -381,6 +381,15 @@
         "v0.20.0"
       ]
     },
+    {
+      "id": "google-cloud-redisw",
```
[the remark this answers is not in the document]
Comment (crwilcox) [reply docs/manifest.json]: fixed.

**Comment by crwilcox** [comment]
@quartzmo , I repushed the tag and this branch to fix the typo.

**Comment by blowmage** [comment]
I am confused why this pull request was opened. Once the tag is added to a repo it is added. There is no review process to approve it.

**Comment by crwilcox** [comment]
@blowmage this likely has to do with my confusion about your process. For both node and python, a 
'release' pr is opened and then merged to master, at which time we make a tag. I walked through the release process and it wasn't clear to me that I was to push straight to master and not PR this.

**Comment by blowmage** [comment]
Okay, we have the option to use a PR to define the release commit, but it should only be tagged once it is in master. I am going to close this PR and make any needed changes to documentation (because of the typo) manually. The original tag already defined a release that published the gem. We shouldn’t be deleting tags that mark a release unless the gem was yanked. I will also take care of adding google-cloud-redis as a dependency in the google-cloud gem, and releasing a new version there as well.

**Pull request closed without merging**